addons/script.module.pil/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.pil"
        name="Python Image Library"
-       version="1.1.7"
+       version="6.2.1"
```

Comment:
Is there a guarantee that at least this version is available?

Reply:
for windows, as we provice that version.
on linux we use whatever the distro provides.
i don't know how it's handled on android / osx

Reply:
Same as for windows, so you can set the version only to 3.0.0.
https://github.com/xbmc/xbmc/blob/5b152cd2f32568e87ee8ba0b3e7f9870028dcca2/tools/depends/target/pythonmodule-pil/Makefile#L9

Reply:
does not compute... pillow 3.0.0 isn't even compatible with python 3.7

Reply:
So you say pil is broken on Android, iOS, macOS and tvOS?

Reply:
It seems, related to here was his Python 3.7 support added on **5.2.0** https://pillow.readthedocs.io/en/stable/releasenotes/5.2.0.html